thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/BaselineEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.thirdeye.rootcause.impl;
+
+import com.linkedin.thirdeye.rootcause.Entity;
+
+
+public class BaselineEntity extends Entity {
+  public static BaselineEntity fromURN(String urn, double score) {
```

Comment:
samples added.
@npawar we could have used a baseline time-range (pseudo-entity) in the search context as well. we went with an entity wrapper to stay consistent with out knowledge graph schema